defaults/test_default.yaml

```diff
@@ -75,3 +75,5 @@ store_cdclog_reader_stats_in_es: false
 region_aware_loader: false
 
 stop_test_on_stress_failure: true
+
+use_legacy_cluster_init: true
```

Comment:
Do we want this to be `true` by default?

Reply:
This set for backward compatibility with all sct tests. For that moment, it allow enable the new cluster init only for specific tests.

Reply:
Ok